### PR TITLE
implement enhancement #69

### PR DESCRIPTION
**Pull request by khatibomar**

### CODE REVIEW

**Comment by olavfosse** [comment]
Hello, could you make it so that when you scroll away from the bottom it stops autoscrolling?

**Comment by khatibomar** [comment]
yes I am working on it.

**Comment by khatibomar** [comment]
I put a hardcoded value inside I dont think it is a wise Idea. I am not sure if it cause bugs under some conditions.

**Comment by khatibomar** [comment]
Yes it cause issues , working on more efficient solution.

File: public/script.js [hunk 1]
```diff
@@ -220,8 +220,9 @@ socket.addEventListener('message', e => {
 		chatMessage.appendChild(chatMessageContent)
 
 		const chatMessages = document.getElementsByClassName('chat-messages')[0]
+		var scrollWasOnbottom = chatMessages.scrollTop+chatMessages.offsetHeight === chatMessages.scrollHeight
```
Comment (olavfosse) [inline public/script.js]: Do not use var. Use const if you can, otherwise use let.

See http://xahlee.info/js/javascript_style_guide.html for why.

**Comment by olavfosse** [comment]
I see this PR has two commits when only one would be more appropriate. You should squash them into one with [git-rebase](https://git-scm.com/docs/git-rebase).

**Comment by khatibomar** [comment]
I noticed a bug when zoom the tab , working on a fix.

**Pull request merged**